pages/nfttest/fetch/index.vue

```diff
@@ -155,6 +155,14 @@ export default class FetchIndex extends Vue {
     }
   }
 
+  get mintQuery() {
+    const query: any = {}
+    if (this.crawledData?.image) {
+      query.ogImageUrl = encodeURIComponent(this.crawledData.image)
```

Comment:
I think you don't need to call `encodeURIComponent` for query, vue-router would handle that